Test/Test/CharOccurrences.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+
+class CharOccurrences {
+	private:
+		const int static possibleChars = 128;
```

Comment:
Since it is const it can be public. There is no sense to incapsulate it into the method